content/en/docs/ambient/usage/l7-features/index.md

```diff
@@ -109,3 +109,5 @@ spec:
       weight: 10
 EOF
 {{< /text >}}
+
+Read more on how to extend waypoints with Wasm plugins [here](/docs/ambient/usage/extend-waypoint-wasm/).
```

Comment:
Why is this added as the last line? This was supposed to be at the extensions section?

Reply:
corrected